lib/endpoints/class-wp-rest-taxonomies-controller.php

```diff
@@ -107,6 +107,7 @@ public function prepare_item_for_response( $taxonomy, $request ) {
 		$data = array(
 			'name'         => $taxonomy->label,
 			'slug'         => $taxonomy->name,
+			'capabilities' => $taxonomy->cap,
```

Comment:
> Does it make sense to white-list the capabilities?

Hm. I'm not sure I understand the use case of whitelisting.

As far as the data itself goes, it's always expected to be an array of key => value pairs.